systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -804,7 +804,7 @@
 
       model.dirty = false;
 
-      model = createMexPointer(model);
+     % model = createMexPointer(model);
```

Comment:
Hi Twan,

Yes, I agree.   I don't think this should be merged until a better solution
is determined.

I am curious if you have intuition as to the bug, though.  If the
parameters are msspoly, then this gets called and breaks things.  However,
according to Russ, this shouldn't get called when a manipulator has
msspoly's.

I don't know where the logic is for where this if statement is though.  I
tried to find it, but couldn't, so either I missed it, or that line is in
the mex itself (which I didn't want to touch).

My intuition is that the bug happens when a manipulator has some bodies
which are all double, and somebodies which are part msspoly, but I have not
yet been able to confirm it.

-Andy S.


On Thu, Jul 30, 2015 at 11:05 PM, Twan Koolen <notifications@github.com>
wrote:

> In systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m
> <https://github.com/RobotLocomotion/drake/pull/1212#discussion_r35943574>:
>
> > @@ -804,7 +804,7 @@
> >
> >        model.dirty = false;
> >
> > -      model = createMexPointer(model);
> > +     % model = createMexPointer(model);
>
> bad change
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/1212/files#r35943574>.
>